scene_id: 0
main_loop_freq: 30.0
unity_render: yes 

fov: 90
width: 256
height: 256
num_camera: 1
camera_names: ["camera_1", "camera_2", "camera_3"]
camera_1:
  relpose_T: [0.0, 0.0, -0.1]
  relpose_R: [0.924, -0.383, 0.0, 0.0, ]

camera_2: 
  relpose_T: [0.0, 0.0, -0.4]
  relpose_R: [1.0, 0.0, 0.0, 0.0]

camera_3:
  relpose_T: [0.0, 0.0, -0.4]
  relpose_R: [0.924,-0.383,  0.0, 0.0]
